README.md

```diff
@@ -0,0 +1,73 @@
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Live Link
+Click here for [live link](https://romantic-williams-bee2ec.netlify.app/)
+
+## Available Scripts
+
+In the project directory, you can run:
+
```

Comment:
- [ ] It would be better if we can add instructions about how to clone the app locally, so non-developers can also work with the repo. It would also be great to include your social media handles, etc. Let us kindly use the [Microverse template](https://github.com/microverseinc/readme-template) as inspiration